### DISCUSSION TITLE
Rasa Sending Double/Duplicate Replies

### DISCUSSION THREAD

**Discussion by ghost** [chatwoot/chatwoot#8001] [Q&A]
### Describe the bug

I have successfully integrated a Rasa chatbot into my Chatwood instance, running on an Ubuntu virtual machine. However, I've encountered an issue where the chatbot is sending duplicate replies for a single chat, as illustrated in the attached image.

![screenshot-2023-09-22-17-52-24](https://github.com/chatwoot/chatwoot/assets/140147735/d442a056-dd61-45a5-877c-c458fc8d4ca7)

Could you please advise if there might be a configuration error or omission on my part?

### To Reproduce

1. Insatll chatwood.
2. Insatlled rasa through https://rasa.com/docs/rasa/installation/environment-set-up.
3. Created and started rasa project, done all configs following the docs at https://github.com/chatwoot/rasa-agent-bot-demo

### Environment

Linux VM

### Cloud Provider

None

### Platform

Browser

### Operating system

Ubuntu 22.04

### Browser and version

Brave Version 1.57.57 Chromium: 116.0.5845.163 (Official Build) (64-bit)

**Comment by pranavrajs** [comment]
Moving this to a discussion as this is not an issue with Chatwoot. @nestdeployments Please check if you calling the APIs twice in your RasaBot.
